app/controllers/meetings_controller.rb

```diff
@@ -55,9 +55,9 @@ def join
   end
 
   def get_occupancy
-    capacity  = Meeting.capacity(params[:id])
+    @max_occupancy = Meeting.capacity(params[:id])
     attending = EmployeeMeeting.attending(params[:id])
-    @current_occupancy = capacity - attending
+    @current_occupancy = @max_occupancy  - attending
   end
 
```

Comment:
no need for max_occupancy to be an instance variable if it is not being used elsewhere in the app